### PR TITLE
Fix/contributing guide

### LINKED ISSUE #51: Contributing
The contributing guide is still a little hard to read, and it does not give accurate information about how to give examples.

In particular, the Relation Style Guide is really the most important part.

Maybe instead of a table, it would be better to have a list or collection of subheadings with explanations?

### PR DESCRIPTION
**Pull request by gconnect**
**What this PR does**
- This fixes #51 
- Change the style guide table to list display
- Added some content

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -29,86 +29,80 @@ Documentation Convention
...
-generate the html files from this automatically.
+When writing please consider your readers, we would like the documentation reader friendly for beginners as well as useful for experts.
 
-When writing please consider your readers, we would like the
-documentation reader friendly for beginners as well as useful for
-experts.
+How to add a new relation
+=========================
+
+Documentation for a new relation should be done after discussing the relation with generally.
```
Comment (goodmami) [inline CONTRIBUTING.rst:39]: I'm not sure exactly what you were going for here. Maybe "after discussing the relation with the group"?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -29,86 +29,80 @@ Documentation Convention
...
+
+- Add an issue suggesting what should be added
+- Add the relations to RELATIONS in ``inventories.py``
+- Add basic information about the relations to ``doc_basic.py``
+- Add documentation to (at least) ``doc_en.py``
+- Check the project doc folder to have access to useful content in the `Euro Wordnet General Documentation <https://globalwordnet.github.io/gwadoc/pdf/EWN_general.pdf>`_ and `Piasecki_Wordnet <https://globalwordnet.github.io/gwadoc/pdf/ Piasecki_Wordnet.pdf>`_  pdf documents.
```
Comment (goodmami) [inline CONTRIBUTING.rst:45]: First, how about "Check the `docs/pdf/` folder...", but also how does having "access to useful content" in these files help with adding a relation? That part isn't clear.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -29,86 +29,80 @@ Documentation Convention
...
-+-----------+-----------------------------------------------------------+
-
-.. _Meat: https://lr.soh.ntu.edu.sg/omw/ili/concepts/77100
-.. _Beef: https://lr.soh.ntu.edu.sg/omw/ili/concepts/77197
-
-
-**Other name in specific Project**
+- **Relation Name** :  The name of the relation
+
+- **Short Definition** : The short definition only shows up when the user’s mouse hover over the relation name, it is a brief introduction of the relation.
```
Comment (goodmami) [inline CONTRIBUTING.rst:53]: The "mouse hover" behavior depends on which application is using gwadoc. Is this for OMW?

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -29,86 +29,80 @@ Documentation Convention
...
+
+1. Use concept A / concept B (A / B) for reference, DO NOT use concept X / concept Y (X /y).
+
+2. Use “A is … of B” as the sentence pattern if possible.
```
Comment (goodmami) [inline CONTRIBUTING.rst:57]: This could benefit from an example.

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -29,86 +29,80 @@ Documentation Convention
...
+
+- **Short Example** : The short example only shows up when the user’s mouse hover over the relation name, it just shows the examples without links to lexicon. At least 2 examples. If there is no example in the docs, go to related resources or wiki to find more.
```
Comment (goodmami) [inline CONTRIBUTING.rst:59]: Ditto about "mouse hover". And what are the related resources or wiki?

**Review by goodmami — COMMENTED** [review]
Just a few thoughts...

File: CONTRIBUTING.rst (hunk 1 above)
Comment (gconnect) [reply CONTRIBUTING.rst:39]: Corrected

File: CONTRIBUTING.rst (hunk 2 above)
Comment (gconnect) [reply CONTRIBUTING.rst:45]: Corrected

File: CONTRIBUTING.rst (hunk 3 above)
Comment (gconnect) [reply CONTRIBUTING.rst:53]: Corrected

File: CONTRIBUTING.rst (hunk 4 above)
Comment (gconnect) [reply CONTRIBUTING.rst:57]: Example has been provided already, or I probably don't understand what you meant

File: CONTRIBUTING.rst (hunk 5 above)
Comment (gconnect) [reply CONTRIBUTING.rst:59]: Corrected.

**Pull request merged**